### ISSUE TITLE
Hide "Reports recorded" in tablet debugger if data recording hasn't yet been activated

### ISSUE THREAD

**Issue by gonX** [OpenTabletDriver/OpenTabletDriver#2000]
## Description
This isn't exactly intuitive despite the title of the textbox.

Setting `Visible = false` in code doesn't reflow the UI, so the fix is less simple than expected.

If reflowing the UI is hard we could probably drop this text box entirely from the UI. We can also make a fixed-width spinner (alternating `/ - \ |`) instead so that the user at least knows something is being logged.

**Comment by Sublimelime** [comment]
I disagree with just hiding it - once you've stopped recording, it's potentially useful to know how many reported you record**ed**, past tense.

Instead, how about hiding it until some reports are recorded, then displaying it (permanently after)? Though tbh this is kinda a non-issue as the text box is pretty small and keeping it as-is doesn't strike me as problematic.

**Comment by gonX** [comment]
Yes, that's what I meant. We should only show it if we have recorded in this session. Thanks for the correction.

It has confused some users before thinking that reports aren't being parsed correctly, which is why I want the change

**Comment by gonX** [comment]
Moving milestone to 0.6.6 because Avalonia branch has this considered.

**Comment by gonX** [comment]
Fixed in #4048
